props/saber_fett263_buttons.h

```diff
@@ -251,10 +251,15 @@ SaberFett263Buttons() : PropBase() {}
 
       // EVENT_PUSH
       if (fabs(mss.x) < 3.0 &&
-          mss.y * mss.y + mss.z * mss.z > 120 &&
+          mss.y * mss.y + mss.z * mss.z > 100 &&
           fusor.swing_speed() < 30 &&
           fabs(fusor.gyro().x) < 10) {
-        Event(BUTTON_NONE, EVENT_PUSH);
+        if (millis() - push_begin_millis_ > 25) {
```

Comment:
This is not what I suggested, the else is in the wrong place.
As written, this will do roughly the opposite to what we want.
push_begin_millis_ will be set to millis() every time that the push detection triggers.
push events will be generated if it's been more than 25 ms since the last push detection, but it will not do anything to filter out very short events.

Reply:
sorry, fixed. did have to lower for push as it's a very short movement as opposed to thrust